application/forms/RotationConfigForm.php

```diff
@@ -199,97 +202,8 @@ public function __construct(int $scheduleId, Connection $db)
-        }
-
-        $formData = [
-            'mode' => $rotation->mode,
-            'name' => $rotation->name,
-            'priority' => $rotation->priority,
-            'schedule' => $rotation->schedule_id,
-            'options' => $rotation->options
-        ];
-        if (! self::EXPERIMENTAL_OVERRIDES) {
-            $formData['first_handoff'] = $rotation->first_handoff;
-        }
-
-        if (self::EXPERIMENTAL_OVERRIDES) {
-            $getHandoff = function (Rotation $rotation): DateTime {
-                switch ($rotation->mode) {
-                    case '24-7':
-                        $time = $rotation->options['at'];
-
-                        break;
-                    case 'partial':
-                        $time = $rotation->options['from'];
-
-                        break;
-                    case 'multi':
-                        $time = $rotation->options['from_at'];
-
-                        break;
-                    default:
-                        throw new LogicException('Invalid mode');
-                }
-
-                $handoff = DateTime::createFromFormat('Y-m-d H:i', $rotation->first_handoff . ' ' . $time);
-                if ($handoff === false) {
-                    throw new ConfigurationError('Invalid date format');
-                }
-
-                return $handoff;
-            };
-
-            $this->previousHandoff = $getHandoff($rotation);
-
-            /** @var ?TimeperiodEntry $previousShift */
-            $previousShift = TimeperiodEntry::on($this->db)
-                ->columns('until_time')
-                ->filter(Filter::all(
-                    Filter::equal('timeperiod.rotation.schedule_id', $rotation->schedule_id),
-                    Filter::equal('timeperiod.rotation.priority', $rotation->priority),
-                    Filter::unequal('timeperiod.owned_by_rotation_id', $rotation->id),
-                    Filter::lessThanOrEqual('until_time', $this->previousHandoff),
-                    Filter::like('until_time', '*')
-                ))
-                ->orderBy('until_time', SORT_DESC)
-                ->first();
-            if ($previousShift !== null) {
-                $this->previousShift = $previousShift->until_time;
-            }
-
-            /** @var ?Rotation $newerRotation */
-            $newerRotation = Rotation::on($this->db)
-                ->columns(['first_handoff', 'options', 'mode'])
-                ->filter(Filter::all(
-                    Filter::equal('schedule_id', $rotation->schedule_id),
-                    Filter::equal('priority', $rotation->priority),
-                    Filter::greaterThan('first_handoff', $rotation->first_handoff)
-                ))
-                ->orderBy('first_handoff', SORT_ASC)
-                ->first();
-            if ($newerRotation !== null) {
-                $this->nextHandoff = $getHandoff($newerRotation);
-            }
-        }
```

Comment:
This should be kept here. It's not used to populate the form's inputs directly and may be of disadvantage in `fetchDbValues` as if called twice, resets the properties.